### ISSUE TITLE
Support ARM

### ISSUE THREAD

**Issue by fdsagizi** [BlindMindStudios/AngelScript-JIT-Compiler#6]
Hi!
It's possible to use JIT on iOS ?

**Comment by TheExDeus** [comment]
How about Raspberry Pi or Jetson? At least RPi is cheap and easy to use. I would love to have the JIT work on it.

**Comment by ThyReaper** [comment]
I'm not familiar with either of those. If they are running x86 processors, it would hopefully be simple to write a device-specific code version of the platform files, which I would gladly accept into the repo.

**Comment by BurningEnlightenment** [comment]
The RPis are ARM based single board computers and I guess the Jetson will be too (even though I never heard of it).

**Comment by TheExDeus** [comment]
Yeah, sorry Raspberry Pi is a very popular ARM PC - http://www.raspberrypi.org/blog/
And Jetson is an ARM PC created by Nvidia - http://www.nvidia.com/object/jetson-tk1-embedded-dev-kit.html

Maybe Jetson isn't a good example, but Raspberry Pi can be bought for 35$, it's very popular and is very easy to use. It was my suggestion for a platform on which to test.

**Comment by quarnster** [comment]
FWIW I wrote an ARM JIT for AS back in 2009, but I never finished it and haven't touched it since neither: https://github.com/quarnster/asarmjit.
